charts/scalardl/templates/ledger/deployment.yaml

```diff
@@ -146,9 +136,7 @@ spec:
               - -addr=localhost:50051
               {{- if .Values.ledger.tls.enabled }}
               - -tls
-              {{- if .Values.ledger.tls.caRootCertSecret }}
-              - -tls-ca-cert=/tls/certs/ca-root-cert.pem
-              {{- end }}
+              - -tls-ca-cert=/tls/scalardl-ledger/certs/ca.crt
```

Comment:
When you enable TLS, the `-tls-ca-cert` option is mandatory. In other words, `{{- if .Values.scalardbCluster.tls.caRootCertSecret }}` does not make sense. So, I removed that unnecessary condition.